go.mod

```diff
@@ -2,7 +2,7 @@ module sigs.k8s.io/cluster-api-operator
 
 go 1.19
```

Comment:
are we planning to bump go version to 1.20 as per https://main.cluster-api.sigs.k8s.io/developer/providers/migrations/v1.4-to-v1.5.html#go-version ?